cpp/benchmarks/join/join_benchmark.cu

```diff
@@ -82,13 +82,27 @@ static void join_benchmark(benchmark::State& state)
 
     // Benchmark the inner join operation
 
+    cudaEvent_t start, stop;
+    cudaEventCreate(&start);
+    cudaEventCreate(&stop);
+
+    float milliseconds;
+
     for (auto _ : state) {
+        cudaEventRecord(start);
```

Comment:
We'll want to create some kind of abstracted wrapper for this kind of thing.

Reply:
@gaohao95 can you update this to use the `cuda_event_timer` object that @hummingtree created in his PR here? https://github.com/rapidsai/cudf/pull/2215/files?file-filters%5B%5D=

You can just merge his PR into yours, and I'll merge his first.

Reply:
Done in https://github.com/rapidsai/cudf/pull/2105/commits/fa29774a4feabd4cce54de0c400bd19fe3cb6618.